src/main/java/edu/kis/powp/jobs2d/command/Loader.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.command;
+
+public interface Loader {
```

Comment:
I do not think the name is appropriate. Maybe CommandReader or CommandImporter? The name of the method should be updated